src/StringSorterAscTest.java

```diff
@@ -0,0 +1,28 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+class StringSorterAscTest {
+	private static StringSorterAsc Sorter;
+	@Test
+	void test() {
+		 
+	       
+	        
+	            ArrayList<Msg> list=new ArrayList<>(5);
+	            list.add(new Msg("aabcda",3255));
```

Comment:
Maybe make some test cases that are easier for a human to read (ie real words), so they could easily see what the function is trying to accomplish in their head.

Reply:
You're right. I've added some explanations.